groundstation/static/js/App.jsx

```diff
@@ -50,11 +50,25 @@ var username = null
 function App() {
   if (isAuthenticated()){
     username = sessionStorage.getItem('username');
+
+    const auth_token = sessionStorage.getItem('auth_token');
+    fetch(`/api/users/${auth_token}`).then(results => {
+        return results.json();
+    }).then(data => {
+        if (data.status === 'success') {
+            setIsAdmin(data.data.is_admin);
+        } else {
+            console.error('Unexpected error occured:');
+        }
+    });
   }
   const classes = useStyles();
   const [open, setOpen] = React.useState(false);
+  const [isAdmin, setIsAdmin] = React.useState(false);
```

Comment:
Also, I think you'd need to move this line before the if statement so that `setIsAdmin()` is a defined function